src/subdomains/generic/forwarding/services/lnurl-forward.service.ts

```diff
@@ -95,16 +101,16 @@ export class LnUrlForwardService {
   }
 
   private getPaymentTransferInfo(params: any): TransferInfo {
-    const isMsat = !params.asset || params.asset === 'MSAT';
+    const isBtc = !params.asset || params.asset === 'BTC';
 
     const amount = params.amount ? Number(params.amount) : 0;
-    const asset = isMsat ? 'BTC' : params.asset;
+    const asset = isBtc ? 'BTC' : params.asset;
     const method = Util.toEnum(Blockchain, params.method) ?? Blockchain.LIGHTNING;
 
     return {
       method: method,
       asset: asset,
-      amount: isMsat ? LightningHelper.msatToBtc(amount) : amount,
+      amount: isBtc ? LightningHelper.msatToBtc(amount) : amount,
       quoteUniqueId: params.quote,
     };

```

Comment:
Does this make sense? If the asset is BTC, convert from MSAT to BTC? I think we can leave this as is, or just remove `|| params.asset === 'MSAT'` (line 98/104)